contracts/test_api_db/test_api_db.cpp

```diff
@@ -0,0 +1,66 @@
+/**
+ *  @file
+ *  @copyright defined in eos/LICENSE.txt
+ */
+#include <eosiolib/eosio.hpp>
+#include "../test_api/test_api.hpp"
+
+#include "test_db.cpp"
+
+extern "C" {
+
+    void init()  {
+
+    }
+
+   void apply( unsigned long long code, unsigned long long action ) {
+
+      WASM_TEST_HANDLER(test_db, primary_i64_general);
+      WASM_TEST_HANDLER(test_db, primary_i64_lowerbound);
+      WASM_TEST_HANDLER(test_db, primary_i64_upperbound);
+      WASM_TEST_HANDLER(test_db, idx64_general);
+      WASM_TEST_HANDLER(test_db, idx64_lowerbound);
+      WASM_TEST_HANDLER(test_db, idx64_upperbound);
+
+      WASM_TEST_HANDLER(test_db, key_str_general);
+#if 0
```

Comment:
since we are deprecating the old API, we can go ahead and remove these stubs too